_posts/2020-12-22-ols.md

```diff
@@ -0,0 +1,76 @@
+---
+site: [pasteur, freiburg, erasmusmc, elixir-it, belgium, genouest]
+title: 'Open Life Science program & the Galaxy community: implication in OLS-2 and invitation to apply to the next cohort'
+tags: training
+supporters:
+- elixir
+- eosc
+- denbi
+---
+
+[Open Life Science (OLS)](https://openlifesci.org/) is an **online mentoring & training program** to help **individuals and stakeholders in research** to become **Open Science ambassadors**. Participants of this program learn essential knowledge required to create, lead, and sustain an Open Science project, connect with members across different communities, backgrounds, and identities by sharing their experiences and expertise.
+
+The first two cohorts have been run in 2020, in which the Galaxy community was successfully involved. Applications are now open for the 3rd cohort (OLS-3) which will take place from **February to May 2021**. This cohort is supported by EOSC-Life. We would love to see a strong representation of the Galaxy community there.
```

Comment:
The first two cohorts ran in 2020, and the Galaxy Project was successfully involved in the second cohort.